api/client/cli.go

```diff
@@ -58,6 +58,8 @@ type DockerCli struct {
 	transport *http.Transport
 }
 
+// Initialize calls the anonymous init function that will setup the configuration for the client
```

Comment:
I think 'anonymous' can be removed, as it can be confused with Go's package init() function